### PR TITLE
[CDU] Improve visuals of CDU Pages

### LINKED ISSUE #2511: [BUG] Simbrief options page returns to wrong mcdu page
**Mod Version**

latest master

**Describe the bug**

Returning page from MCDU Menu -> Options -> AOC -> Simbrief Page is MCDU Menu.
It should return to its father, in this case AOC.

**To Reproduce**

1.
2.
3.

**Expected behavior**

Returns to MCDU Menu Page

**Actual behavior**

Returns to AOC Options Page

**References**

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub): DerL30N#3751

### PR DESCRIPTION
**Pull request by MisterChocker**
Fixes #2511

### The following FMGC Pages have been text indent and color adjusted, I may have also changed some text content. e.g. A/C Status Page Header
- DIR
- PROG and all sub menus
- INIT-A
- INIT-B
- DATA and all sub menus
- F-PLN
- FUEL PRED
- VERT REV
- Depature
- Approach
- Airway
- Data Index
- SEC FPLN
- Airport Monitor
- Ident
- Hold
- IRS
- Navaid
- Pilot Waypoints
- PosFrozen
- PosMon
- SelectedNavaids
- Waypoints
- Options -> AOC -> Simbrief (bug fixes and improvements as well)

## Screenshots (if necessary)

Before:
image
image
After:
image
image

## References

Honeywell Pegasus Step 1A (Rev 0), 2009 edition manual

## Additional context

Discord username (if different from GitHub): DerL30N#3751

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by wpine215** [comment]
Looks like the stored waypoint text above the "DELETE ALL" field is too close - is there any way to adjust the margins/padding a bit so it's consistent with everything else?

**Comment by MisterChocker** [comment]
> Looks like the stored waypoint text above the "DELETE ALL" field is too close - is there any way to adjust the margins/padding a bit so it's consistent with everything else?

tbh, no clue why it's even the way it is, seems buggy. But managed to fix it, just pls don't look how I fixed it xD

**Comment by Noxleee** [comment]
Category: QA Tester
Name: Noxlee #7727
Date of testing: Current date 17/12/2020
Version of the sim: 1.11.7.0
PR Tested: #2475 
Object of test: MCDU Pages

Testing technique: Checked pages mentioned in the PR description
Steps you've made to test the Pull Request:
1. Load AC
2. Checked Pages before Programming FMC
3. Checked Pages after Programming FMC
4. Test route:  EDDL/23L DODE9T DODEN Y852 EDISA Y173 PABLA T715 RIGVI LUME8K LFSB/15 (Did not flown)

Notes: 

- On the DATA page, there are a few menu points without function 
-> recommendation: grey colour like other not working functions in the mcdu

Issues:
- Flight plan: SID/STAR is recognised as UNDEFINED
->SID: 
grafik
->STAR:
grafik

- Departure Runway and STAR stay yellow after insert, same on Arrival page:
-> DEP:
grafik
-> ARR:
grafik



Overall Rating: To be improved
Conclusions of the testing: Found two Issues described above and made one recommendation in the notes. All other pages are looking good so far nothing seems to be misaligned.

**Comment by MisterChocker** [comment]
Waiting for #2263

**Comment by wpine215** [comment]
> 
> 
> Category: QA Tester
> Name: Noxlee #7727
> Date of testing: Current date 17/12/2020
> Version of the sim: 1.11.7.0
> PR Tested: #2475
> Object of test: MCDU Pages
> 
> Testing technique: Checked pages mentioned in the PR description
> Steps you've made to test the Pull Request:
> 
>     1. Load AC
> 
>     2. Checked Pages before Programming FMC
> 
>     3. Checked Pages after Programming FMC
> 
>     4. Test route:  EDDL/23L DODE9T DODEN Y852 EDISA Y173 PABLA T715 RIGVI LUME8K LFSB/15 (Did not flown)
> 
> 
> Notes:
> 
>     * On the DATA page, there are a few menu points without function
>       -> recommendation: grey colour like other not working functions in the mcdu
> 
> 
> Issues:
> 
>     * Flight plan: SID/STAR is recognised as UNDEFINED (Stock database -> no navigraph)
>       ->SID:
>       grafik
>       ->STAR:
>       grafik
> 
>     * Departure Runway and STAR stay yellow after insert, same on Arrival page:
>       -> DEP:
>       grafik
>       -> ARR:
>       grafik
> 
> 
> Overall Rating: To be improved
> Conclusions of the testing: Found two Issues described above and made one recommendation in the notes. All other pages are looking good so far nothing seems to be misaligned.

Can you confirm the issues you mentioned don't exist when using the current development version without this PR?

**Comment by Noxleee** [comment]
I tested the current dev. version. No issues there. Seems like the PR tries to find a name for the SID and STAR but can't find it.

**Review by myselfaaryan — CHANGES_REQUESTED** [review]

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IdentPage.js [hunk 1]
```diff
@@ -5,17 +5,17 @@ class CDUIdentPage {
         mcdu.page.Current = mcdu.page.IdentPage;
         mcdu.activeSystem = 'FMGC';
         mcdu.setTemplate([
-            ["A320-200"],
-            ["ENG"],
+            ["A320-251N"],
```
[the remark this answers is not in the document]
Comment (MisterChocker) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IdentPage.js:8]: thx for the heads up, fyi all changes I did where done with Honeywell manual.

**Review by wpine215 — APPROVED** [review]

**Comment by Aarondodd88** [comment]
Awesome! Has anyone noticed how "NEON" the font colours are on the mcdu?.  FFA320 the only way i can match the same colours is by turning the brightness 100%. These colours used seem to match the font colour in the TolissA320. FBWA320 No brightness changes the font colour on the mcdu, so always have such a bright "NEON" font colour. The font may be the correct colour but for some reason it becomes to fluorescent

**Comment by MisterChocker** [comment]
No changes to color definition in this pr. Only text alignment corrections and some small other text changes.

**Pull request merged**